### ISSUE TITLE
Output columns mixed up during .hdf5 to .bin conversion ?

### ISSUE THREAD

**Issue by GayatriChandran** [ZhuangLab/storm-analysis#43]
I'm having trouble rendering daostorm's output in Insight3. Even after setting the counts to photons conversion factor as 1, the rendered gaussians don't look sharp. I was wondering if this is because some parameters are not being assigned to the correct columns during `.hdf5` to `.bin` conversion ? For example, in `storm_analysis/sa_library/i3dtype.py`,  `createFromMultiFit()` :
 ```python
 if "error" in peaks:
      setI3Field(i3data, 'i', peaks["error"])
```
Here, `i` is supposed to be the intensity, or the sum of pixel values in the fitting region minus background (in ADU). However, `error` in .hdf5 format is the fitting error chi-square value.
It looks like `error` instead of `sum` (in ADU) is being assigned to `i` ?
Similarly, `sum` appears to be assigned to `a` which is area of integration ? 
```python
if "sum" in peaks:
     setI3Field(i3data, 'a', peaks["sum"])
```

**Comment by HazenBabcock** [comment]
I think I made that choice because I was interested in being able to filter based on `error` in Insight3, however it has been a while since I've used that software. I do remember that I usually used the fix width Gaussian rendering so I was untroubled by the issues that you are reporting. The Insight3 `a` parameter is I think the area of Gaussian with height `h` and width `w`. If you find that you get the rendering that you want using `sum` for `i` then go ahead.

**Comment by GayatriChandran** [comment]
I realized that Insight3 actually uses the `a` column for filtering based on photons. So I was looking at the correct rendering after all. However, the 'normalized Gaussian' rendering mode uses `bg` and `w` for deciding how to display each localization. When I set the `bg` column to 0, I get a sharper image. With `w` equals `0`, it becomes a histogram.
